### ISSUE TITLE
[staging] Unable to complete last step of installation

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#16228]
Installing staging version as of https://github.com/joomla/joomla-cms/commit/d57a8e51a2082e102e9a2b5ce5526584d5ab6142

On the final page, it starts to install and then redirects to the final page.

### Actual result
```
[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:48: parser error : Opening and ending tag mismatch: fieldset line 23 and field in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 				&lt;/field&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 				        ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:49: parser error : expected '&gt;' in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 			&lt;/fieldset&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 			        ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:50: parser error : Opening and ending tag mismatch: config line 21 and fields in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 		&lt;/fields&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 		         ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:51: parser error : Opening and ending tag mismatch: extension line 2 and config in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 	&lt;/config&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 	         ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:52: parser error : Extra content at the end of the document in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): &lt;/extension&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 1997

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:48: parser error : Opening and ending tag mismatch: fieldset line 23 and field in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 				&lt;/field&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 				        ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:49: parser error : expected '&gt;' in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 			&lt;/fieldset&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 			        ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:50: parser error : Opening and ending tag mismatch: config line 21 and fields in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 		&lt;/fields&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 		         ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:51: parser error : Opening and ending tag mismatch: extension line 2 and config in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 	&lt;/config&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): 	         ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): file:///C:/xampp/htdocs/joomla-cms/plugins/fields/radio/radio.xml:52: parser error : Extra content at the end of the document in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): &lt;/extension&gt; in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215

[24-May-2017 06:50:23 Europe/Berlin] PHP Warning:  simplexml_load_file(): ^ in C:\xampp\htdocs\joomla-cms\libraries\cms\installer\installer.php on line 2215
```

**Comment by franz-wohlkoenig** [comment]
is this on 4.0?

**Comment by Quy** [comment]
No. V3.7.3-dev

**Comment by franz-wohlkoenig** [comment]
Can't confirm. Installation of staging works well.

**Comment by brianteeman** [comment]
Sounds like a bug in one of the xml pull requests I did that has been
merged. I will check it later today

**Comment by infograf768** [comment]
I corrected one of them
https://github.com/joomla/joomla-cms/commit/15c4ccca3e56bac6765c65320621a4f075d2f260

**Comment by infograf768** [comment]
one / too much for the field

**Comment by rdeutz** [comment]
fixed wth d7aaddf7ba5b449b6108d4853f0514dc93f38605

**Comment by brianteeman** [comment]
Thanks beat me to it. Still too early in the morning for me
